mini_pupper_driver/launch/lidar_ld06.launch.py

```diff
@@ -31,7 +33,7 @@ def generate_launch_description():
                 {'product_name': 'LDLiDAR_LD06'},
                 {'topic_name': 'scan'},
                 {'frame_id': 'lidar_link'},
-                {'port_name': '/dev/ttyAMA1'},
+                {'port_name': lidar_port}
```

Comment:
I think we need to the same LaunchConfiguration and DeclareLaunchArgument here in this file. 

     lidar_port = LaunchConfiguration("lidar_port")
     lidar_port_launch_arg = DeclareLaunchArgument(
         name='lidar_port',
         description='The serial port for the lidar sensor'
     )